Directory.Packages.props

```diff
@@ -6,11 +6,11 @@
     <PackageVersion Include="Microsoft.Azure.Functions.Worker.Extensions.Abstractions" Version="1.2.0" />
     <PackageVersion Include="Microsoft.ApplicationInsights" Version="2.21.0" />
     <PackageVersion Include="Microsoft.Azure.WebJobs" Version="3.0.37" />
-    <PackageVersion Include="Microsoft.Data.SqlClient" Version="5.1.1" />
+    <PackageVersion Include="Microsoft.Data.SqlClient" Version="5.0.1" />
     <PackageVersion Include="Microsoft.SourceLink.GitHub" Version="1.1.1" />
     <PackageVersion Include="morelinq" Version="3.4.2" />
     <PackageVersion Include="System.Drawing.Common" Version="7.0.0" />
-    <PackageVersion Include="System.Runtime.Caching" Version="7.0.0" />
+    <PackageVersion Include="System.Runtime.Caching" Version="5.0.0" />
```

Comment:
Can we just revert to 6.0.0 for this instead? MDS 5.1.1 says it just requires this >= 6.0.0 so it would be nice to keep the MDS update if possible.

https://www.nuget.org/packages/Microsoft.Data.SqlClient/5.1.1#dependencies-body-tab

And then we should figure out if we can make a test to catch this kind of failure in the future.